www/src/js/data/venues.json

```
@@ -3686,5 +3686,37 @@
       "x": 103.77160638570787,
       "y": 1.2994984664852203
     }
+  },
+  "SPSWETLAB": {
+    "roomName": "SPSWETLAB",
+    "floor": 2,
+    "location": {
+      "x": 103.78030210733415,
+      "y": 1.296855328454714
+    }
+  },
+  "BIZ2-0413B": {
+    "roomName": "Seminar Room 04-13B",
+    "floor": 4,
+    "location": {
+      "x": 103.77504911418305,
+      "y": 1.2933189348382488
+    }
+  },
+  "BIZ2-0413A": {
+    "roomName": "BIZ2-0413A",
```

```suggestion
    "roomName": ""Seminar Room 04-13A",
```

For consistency with the one above